includes/payment-methods/class-wc-stripe-upe-payment-method-afterpay-clearpay.php

```diff
@@ -70,27 +70,11 @@ public function __construct() {
 	 */
 	public function get_title( $payment_details = false ) {
 		if ( $this->is_gb_country() ) {
-			return __( 'Clearpay', 'woocommerce-gateway-stripe' );
+			$this->title = __( 'Clearpay', 'woocommerce-gateway-stripe' );
 		}
-		return __( 'Afterpay', 'woocommerce-gateway-stripe' );
-	}
+		$this->title = __( 'Afterpay', 'woocommerce-gateway-stripe' );
 
-	/**
-	 * Return the gateway's description.
-	 *
-	 * @return string
-	 */
-	public function get_description( $payment_details = false ) {
-		if ( $this->is_gb_country() ) {
-			return __(
-				'Allow customers to pay over time with Clearpay.',
-				'woocommerce-gateway-stripe'
-			);
-		}
-		return __(
-			'Allow customers to pay over time with Afterpay.',
-			'woocommerce-gateway-stripe'
-		);
```

Comment:
Per https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3282, we no longer display the default description if a custom description is not provided. We delete this method override as it is no longer necessary.